getting-started/windows-calico/openshift/installation.md

```diff
@@ -167,61 +185,56 @@ $ ./wni aws create \
-   ----                      --------------------                    ------- ------       ----------             ---------
-   vEthernet (nat)           Hyper-V Virtual Ethernet Adapter             10 Up           00-15-5D-BA-0F-68        10 Gbps
-   Ethernet 2                Amazon Elastic Network Adapter                6 Up           0A-4F-1F-C4-E3-49        25 Gbps
-   vEthernet (Ethernet 2)    Hyper-V Virtual Ethernet Adapter #2          15 Up           0A-4F-1F-C4-E3-49        25 Gbps
+   > **Note**: Get the Kubernetes version with `oc version` and use only the major, minor, and patch version numbers. For example from a cluster that returns:
    ```
-
-   Then, add the route using the interface index:
-   ```powershell
-   PS C:\> New-NetRoute -DestinationPrefix 169.254.169.254/32 -InterfaceIndex <interface_index>
+   $ oc version
+   Client Version: 4.5.3
+   Server Version: 4.5.14
+   Kubernetes Version: v1.18.3+5302882
    ```
+   You will use `1.18.3`:
+   {: .alert .alert-info}
 
 1. Install and start kube-proxy service. Execute following powershell script/commands.
 
    ```powershell
    C:\CalicoWindows\kubernetes\install-kube-services.ps1 -service kube-proxy
    Start-Service -Name kube-proxy
    ```
+
 1. Verify kube-proxy service is running.
 
    ```powershell
-   PS C:\> Get-Service -Name kube-proxy
-
-   Status   Name               DisplayName
-   ------   ----               -----------
-   Running  kube-proxy         kube-proxy service
+   Get-Service -Name kube-proxy
    ```
 
 #### Configure kubelet
 
-Copy the previously downloaded file `wmcb.exe` and your worker's ignition file (worker.ign) to the Windows node.
-To download the worker.ign:
+From the Windows node, download the Windows Machine Config Bootstrapper `wmcb.exe` that matches your OpenShift minor version from {% include open-new-window.html text='Windows Machine Config Bootstrapper releases' url='https://github.com/openshift/windows-machine-config-bootstrapper/releases' %}. For example, for OpenShift 4.5.x:
 
-```shell
-apiserver=$(oc get po -n  openshift-kube-apiserver -l apiserver=true --no-headers -o custom-columns=":metadata.name" | head -n 1)
-oc -n openshift-kube-apiserver exec ${apiserver} -- curl -ks https://localhost:22623/config/worker > worker.ign
+```powershell
+curl https://github.com/openshift/windows-machine-config-bootstrapper/releases/download/v4.5.2-alpha/wmcb.exe -o c:\wmcb.exe
+```
+
+Next, we will download the the `worker.ign` file from the API server:
+
+```powershell
+$apiServer = c:\k\kubectl --kubeconfig c:\k\config get po -n  openshift-kube-apiserver -l apiserver=true --no-headers -o custom-columns=":metadata.name" | select -first 1
+c:\k\kubectl --kubeconfig c:\k\config -n openshift-kube-apiserver exec $apiserver -- curl -ks https://localhost:22623/config/worker > c:\worker.ign
+((Get-Content c:\worker.ign) -join "`n") + "`n" | Set-Content -NoNewline c:\worker.ign
```

Comment:
This keeps the user on the Windows node to configure kubelet, rather than having them go to the Linux node to generate the worker ignition file.